dense_retriever.py

```diff
@@ -234,9 +234,9 @@ def main(args):
     ctx_files_pattern = args.encoded_ctx_file
     input_paths = glob.glob(ctx_files_pattern)
 
-    index_path = "_".join(input_paths[0].split("_")[:-1])
-    if args.save_or_load_index and os.path.exists(index_path):
-        retriever.index.deserialize(index_path)
+    index_path = "_".join(input_paths[-1].split("_")[:-1])
```

Comment:
Sorry, what is the purpose of changing 0 to -1 here?

Reply:
It is to get ctx_files_pattern! I attach related part in my configuration.
>>> ctx_files_pattern = 'dpr_ctx*'
>>> input_paths = glob.glob(ctx_files_pattern)
>>> input_paths
['dpr_ctx.index.dpr', 'dpr_ctx.index_meta.dpr', 'dpr_ctx_0']
>>> index_path = "_".join(input_paths[0].split("_")[:-1])
>>> index_path
'dpr'
>>> index_path_modified = "_".join(input_paths[-1].split("_")[:-1])
>>> index_path_modified
'dpr_ctx'